https://issues.redhat.com/browse/ACM-12605 (#6652)
open-cluster-management/rhacm-docs: observability/customize_observability.adoc

[#adding-custom-metrics]
== Adding custom metrics

> //I mention managed clusters here, should this be changed to hub clusters? Mentioning "managed cluster" here and in the note seem repetitive | MJ | 07/11 

Author: @douglascamata ^

Reviewer: I left some ideas based on the comment in the lines where it is applicable.

> Add metrics to the `metrics_list.yaml` file to collect from managed clusters. Complete the following steps:

Reviewer: I think this line here could use some improvement in wording, which can later be reflected on the new one being added.

```suggestion
Add metrics to the `metrics_list.yaml` file to start collect them from all your managed clusters. Complete the following steps:
```

oc apply -n open-cluster-management-observability -f observability-metrics-custom-allowlist.yaml
----

+

> *Note:* The metrics collector also collects allowlist from your managed cluster.

Author: ```suggestion
*Note:* The metrics collector also collects the allowlist from your managed cluster.
```

> *Note:* The metrics collector also collects the allowlist from your managed cluster.

Reviewer: I think we can present this information after teaching the users about the custom metrics configuration through the Hub. 

We then mention that a ConfigMap with the same name, namespace, and structure can be created directly in managed cluster to customize their metrics one by one.

Author: Thanks for your feedback, making updates

> *Note:* The metrics collector collects separate allowlist if it exists in your managed cluster.

Author: ```suggestion
*Note:* The metrics collector also collects allowlist from your managed cluster.
```

<2> *Optional:* Enter only one value for the `expr` and `record` parameter pair to define the query expression. The metrics are collected as the name that is defined in the `record` parameter from your managed cluster. The metric value returned are the results after you run the query expression.
+
You can use either one or both of the sections. For user workload metrics, see the _Adding user workload metrics_ section.
+

> *Note:* The metrics collector also collects the allowlist from your managed cluster. You can create the same YAML directly on your managed cluster to customize the metrics.

Reviewer: After applying my suggested change from the other comment (no need to be verbatim, of course), I think this could become: 

```suggestion
*Note:* The custom metrics allowlist can also be customized differently per managed cluster instead of applied across the whole fleet. You can create the same YAML directly on your managed cluster to customize it.
```
